python/python/tests/tests_server_modules/search/test_ft_search.py

```diff
@@ -0,0 +1,161 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+
+import json as OuterJson
```

Comment:
OuterJson?  maybe StdJson instead?  Or alias the server_modules.json to `Glidejson`

Reply:
Updated, changed json commands import to `GlideJson`